commands/out.go

```diff
@@ -248,8 +258,15 @@ func createAuth(req resource.OutRequest) *authn.Basic {
 func aliasesToBump(req resource.OutRequest, repo name.Repository, ver *semver.Version) ([]name.Tag, error) {
 	variant := req.Source.Variant
 
-	opts := []remote.Option{}
-	opts = append(opts, remote.WithAuth(createAuth(req)))
```

Comment:
Can this `createAuth` function be removed now?

Reply:
`createAuth ` is still used in the `signImages` function. The `gcr.NewTrustedGcrRepository()` function requires authorization to be passed.